cmd/tempo-cli/main.go

```diff
@@ -22,8 +22,11 @@ const (
 	bloomFilePrefix = "bloom-"
 )
 
+var generateJsonnet = false
```

Comment:
Instead of a global var this could be an option in `analyseBlockCmd`.  Or do you see it being useful for other commands?